src/Appwrite/Utopia/Response/Model/User.php

```diff
@@ -18,28 +18,55 @@ public function __construct()
                 'example' => '5e5ea5c16897e',
             ])
             ->addRule('$createdAt', [
-                'type' => self::TYPE_INTEGER,
-                'description' => 'User creation date in Unix timestamp.',
-                'default' => 0,
-                'example' => 1592981250,
+                'type' => self::TYPE_DATETIME,
+                'description' => 'User creation date in Datetime.',
+                'default' => '',
+                'example' => self::TYPE_DATETIME_EXAMPLE,
             ])
             ->addRule('$updatedAt', [
-                'type' => self::TYPE_INTEGER,
-                'description' => 'User update date in Unix timestamp.',
-                'default' => 0,
-                'example' => 1592981250,
+                'type' => self::TYPE_DATETIME,
+                'description' => 'User update date in Datetime.',
+                'default' => '',
+                'example' => self::TYPE_DATETIME_EXAMPLE,
             ])
             ->addRule('name', [
                 'type' => self::TYPE_STRING,
                 'description' => 'User name.',
                 'default' => '',
                 'example' => 'John Doe',
             ])
+            ->addRule('password', [
+                'type' => self::TYPE_STRING,
+                'description' => 'Hashed user password.',
+                'default' => '',
+                'example' => '$argon2id$v=19$m=2048,t=4,p=3$aUZjLnliVWRINmFNTWMudg$5S+x+7uA31xFnrHFT47yFwcJeaP0w92L/4LdgrVRXxE',
+            ])
+            ->addRule('hash', [
+                'type' => self::TYPE_STRING,
+                'description' => 'Password hashing algorithm.',
+                'default' => '',
+                'example' => 'argon2',
+            ])
+            ->addRule('hashOptions', [
+                'type' => [
+                    Response::MODEL_ALGO_ARGON2,
+                    Response::MODEL_ALGO_SCRYPT,
+                    Response::MODEL_ALGO_SCRYPT_MODIFIED,
+                    Response::MODEL_ALGO_BCRYPT,
+                    Response::MODEL_ALGO_PHPASS,
+                    Response::MODEL_ALGO_SHA,
+                    Response::MODEL_ALGO_MD5, // keep least secure at the bottom. this order will be used in docs
+                ],
+                'description' => 'Password hashing algorithm configuration.',
+                'default' => [],
+                'example' => new \stdClass(),
+                'array' => false,
+            ])
```

Comment:
@Meldiron Should these be here? I don't think we should return this info in the response model